Loops/Fix infinite execution/tests/test_task.py

```diff
@@ -0,0 +1,20 @@
+import unittest
+import contextlib
+import io
+
+import timeout_decorator
```

Comment:
I've got this while running the checker

Error
Traceback (most recent call last):
  File "C:\Users\Igor.Gerasimov\introduction_to_python\lib\site-packages\timeout_decorator\timeout_decorator.py", line 75, in new_function
    old = signal.signal(signal.SIGALRM, handler)
AttributeError: module 'signal' has no attribute 'SIGALRM'

Something seems off with the lib

Reply:
Seems like we have to look for a workaround